Source/KerbalCombatSystems/RocketTargeting.cs

```diff
@@ -259,14 +264,23 @@ private IEnumerator FireRocket()
 
         private void NextRocket()
         {
-            decouplers = FindDecouplerChildren(part.parent, "Weapon", true);
+            decouplers = FindDecouplerChildren(part.parent, "Default", true);
             if (decouplers.Count < 1)
             {
                 controller.canFire = false;
                 return;
             }
 
             decoupler = decouplers.Last();
+            AssignReference(decoupler.part.GetReferenceTransform(), AimVector);
+        }
+
+        private void AssignReference(Transform Position, Vector3 Direction)
```

Comment:
Are you sure this works? I think this will just rotate the decoupler towards aimVector.

Reply:
It does neither at the moment and acts identically to initial beta release just using decoupler up vector based on tests. The goal is to change the decouplers part reference to align parallel with aim vector.

Reply:
This function visibly rotates the decoupler. Transform is a reference type so when you change the rotation of the transform on line 282 you are rotating the decoupler. Transforms have to be attached to a GameObject so you'd need to create a new gameobject if you wanted a new transform to work with.

I'd suggest removing the call to this function for now and working out something else to get the ship to aim using the thrust vector, possibly after we have a custom PID (which will be some time) because then we can use quaternions to aim and won't need to set reference transforms.

Reply:
Version for now comments out the call, lack of function excludes a lot of different weapon and missile designs so is considered too high a priority to leave until custom PID.